tendrl/bridge_common/tests/sds_state_node.yaml

```diff
@@ -0,0 +1,74 @@
+---
+tendrl_schema_version: 1
+valid_objects:
+- general
+
+object_details:
+  tendrl_context:
+    enabled: True
+    value: ~
+    attrs:
+      sds_nvr:
+        type: String
+        help: "tendrl-node-agent-3.2"
+  general:
+      enabled: True
+      attrs:
+         list:
+           type: List
+         name:
+            type: String
+         disable_gpg_check:
+            type: Boolean
+         enablerepo:
+            type: List
+         disablerepo:
+            type: List
+
+      atoms:
+            dnf:
+              uuid: "874dd5ad-3138-4e8a-86d3-ec39787a9020"
+              name: "dnf"
+              description: "Manages packages with the dnf package manager"
+              run: "atoms.dnf.run.run(*args, **kwargs)"
```

Comment:
This is not the exact value to be put for run. It should be some atom and start() from that